### ISSUE TITLE
Fails when `maxAge` is too large

### ISSUE THREAD

**Issue by vwong** [DAlperin/remix-redis-session#4]
Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂

When I tried setting a `maxAge` that is sufficiently large, Redis complains that the expiry duration needs to be in [integer] seconds. I believe the problem is due to the use of floating point numbers. This patch fixed it for me, so it'll be great if you can apply this or similar to the code.

```diff
diff --git a/node_modules/remix-redis-session/dist/index.js b/node_modules/remix-redis-session/dist/index.js
index eefa741..a2b73d4 100644
--- a/node_modules/remix-redis-session/dist/index.js
+++ b/node_modules/remix-redis-session/dist/index.js
@@ -25,7 +25,7 @@ const expiresToSeconds = (expires) => {
     const now = new Date();
     const expiresDate = new Date(expires);
     const secondsDelta = (expiresDate.getTime() - now.getTime()) / 1000;
-    return secondsDelta < 0 ? 0 : secondsDelta;
+    return secondsDelta < 0 ? 0 : Math.round(secondsDelta);
 };
 function createRedisSessionStorage({ cookie, options }) {
     let redis;
```

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by kstoddard** [comment]
I am hitting the same issue fairly consistently setting the `maxAge` to two weeks. I came here to post an issue but one has already be posted! I fixed the same way locally.

**Comment by DAlperin** [comment]
Fix is in v0.0.12
